### ISSUE TITLE
type 'NotificationBuilder' has no member 'jsonDataToPushNotification'

### ISSUE THREAD

**Issue by mario202k** [rafaelsetragni/awesome_notifications#158]
``` 
/Users/mario/Downloads/flutter/.pub-cache/hosted/pub.dartlang.org/awesome_notifications-0.0.6+4/ios/Classes/lib/SwiftAwesomeNotificationsPlugin.swift:299:75: error: type 'NotificationBuilder' has no member 'jsonDataToPushNotification'
            guard let pushNotification:PushNotification = NotificaionBuilder.jsonDataToPushNotification(jsonData: arguments)
```
Am I the only one on that?

**Comment by JobMoll** [comment]
Nope i get the same still looking for a solution.

`   /Users/jobmoll/.pub-cache/hosted/pub.dartlang.org/awesome_notifications-0.0.6+4/ios/Classes/lib/notifications/NotificationBuilder.swift:292:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 12.0, *) {
               ^
    /Users/jobmoll/.pub-cache/hosted/pub.dartlang.org/awesome_notifications-0.0.6+4/ios/Classes/lib/notifications/NotificationBuilder.swift:11:14: note: enclosing scope here
    public class NotificationBuilder {
                 ^
    /Users/jobmoll/.pub-cache/hosted/pub.dartlang.org/awesome_notifications-0.0.6+4/ios/Classes/lib/SwiftAwesomeNotificationsPlugin.swift:299:75: error: type 'NotificationBuilder' has no member 'jsonDataToPushNotification'
            guard let pushNotification:PushNotification = NotificationBuilder.jsonDataToPushNotification(jsonData: arguments)
                                                          ~~~~~~~~~~~~~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/jobmoll/.pub-cache/hosted/pub.dartlang.org/awesome_notifications-0.0.6+4/ios/Classes/lib/extensions/UNMutableNotificationContentExtension.swift:25:12: warning: unnecessary check for 'iOS'; enclosing scope ensures guard will always be true
            if #available(iOS 12.0, *) {
               ^
`

**Comment by rafaelsetragni** [comment]
The NotificationBuilder wasnt updated on last release by accident.
Here is the missing code:

```Swift

    
    public static func jsonDataToPushNotification(jsonData:[String : Any?]?) -> PushNotification? {
        if(jsonData?.isEmpty ?? true){ return nil }
        
        let pushNotification:PushNotification? = PushNotification().fromMap(arguments: jsonData!) as? PushNotification
        return pushNotification
    }
```

Sorry, my bad. I gonna update it officially on next release.
